<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>632</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Fenetre principale */
QFrame#MainWindow
{
	background-color:	#C4D1EC;
	border-style:			solid;
	border-color:			#000000;
	border-width:			2px;
}

/* Barre de boutons */
QFrame#frameButtons
{
	background-color:	#535353;
	border-style:			solid;
	border-color:			#000000;
	border-width:			0px 2px 0px 0px;
}

/* Barre de titre */
QFrame#frameTitle
{
	background-color:	#535353;
	border-style:			solid;
	border-width:			0px 0px 2px 0px;
	border-color:			#000000;
}

QFrame#frameTitle &gt; QLabel
{
	color:					#FFFFFF;
	font-weight:		bold;
	font-style:			italic;
	font-size:				14pt;
	font-family:			&quot;Verdana&quot;;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameButtons">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonBooking">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>Planning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClients">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>Clients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBills">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>Factures</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameContent">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameTitle">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="ClientList" name="clientList"/>
        <widget class="BookingList" name="bookingList"/>
        <widget class="QFrame" name="billList"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClientList</class>
   <extends>QFrame</extends>
   <header>Client.ClientList</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookingList</class>
   <extends>QFrame</extends>
   <header>Booking.BookingList</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../loteliQt.qrc"/>
 </resources>
 <connections/>
</ui>
